packages/vscode-extension/src/common/Project.ts

```diff
@@ -124,6 +124,7 @@ export interface ProjectInterface {
   restart(forceCleanBuild: boolean): Promise<void>;
   goHome(homeUrl: string): Promise<void>;
   selectDevice(deviceInfo: DeviceInfo): Promise<boolean>;
+  updateSelectedDevice(deviceInfo: DeviceInfo): void;
```

Comment:
how is this different from `selectDevice`defined above ??

Reply:
`selectDevice` changes active device, so sets different project status and starts a new device session, while updateSelectedDevice simply updates deviceInfo in updateProjectState with changed device name

Reply:
can we only update the selected device?